docs/architecture/_index.md

```diff
@@ -0,0 +1,188 @@
+- Captures and stores data
+
+When `viam-server` can connect to the cloud, it also:
+
+- Automatically pulls configuration updates you make in the Viam app
+- Gets new versions of software packages
+- Uploads and syncs image and sensor data
+- Allows you to remotely control your machine from the Viam app
+
+`viam-server` can use the internet (WAN) or local networks (LAN) to establish peer-to-peer connections between two {{< glossary_tooltip term_id="machine" text="machines" >}}, or to a client application.
+
+## Components, services, modules
+
+A [_component_](/components/) represents a physical piece of hardware in your {{< glossary_tooltip term_id="machine" text="machine" >}}, and the software that directly supports that hardware.
+
+A [_service_](/services/) is a software package that makes it easier to add complex capabilities such as motion planning or object detection to your machine.
+
+Viam has many built-in components and services that run within `viam-server`.
+
+A [_modular resource_](/registry/) is a custom component or service, not built into `viam-server` but rather provided by a _module_ that you or another user have created.
+A module runs in parallel to `viam-server` on your machine, communicating over UNIX sockets, and `viam-server` manages its lifecycle.
+
+{{<imgproc src="/viam/machine-components.png" resize="x1100" declaredimensions=true alt="Machine structure" style="max-width:600px" >}}
+
+{{< expand "Click for an example" >}}
+
+Imagine you have a wheeled rover with two motors, a GPS unit, and a camera, controlled by a single-board computer (SBC) such as a Raspberry Pi.
+
+The motors, GPS, and camera each require software drivers so that signals can be sent to or from them from your board.
+These drivers are called _{{< glossary_tooltip term_id="component" text="components" >}}_.
+There is also a _component_ for the board itself that allows Viam software to communicate with the pins on the board.
+If you configure a [base component](/components/base/), you can specify the size of the wheels attached to the motors, and how wide the rover base is, so `viam-server` can calculate how to coordinate the motion of the rover base.
+
+If your rover includes a piece of hardware (such as a particular sensor) that is not yet supported in Viam, write a {{< glossary_tooltip term_id="module" text="module" >}} to integrate it into your machine.
+
+Say you want your machine to navigate intelligently between GPS coordinates.
+Instead of writing code from scratch, use Viam's built-in navigation service by adding it to your configuration, specifying how the GPS and any additional movement sensors are oriented with respect to your hardware.
+Since your configuration now includes information about the wheeled base of your rover as well as how that base is oriented in relation to your GPS and other sensors, `viam-server` can use the motion and navigation services to calculate how much power to send to each motor to get it to a point on the map.
```

Comment:
It did already for at least a paragraph and you just added a different service so I think remove the now
```suggestion
Since your configuration includes information about the wheeled base of your rover as well as how that base is oriented in relation to your GPS and other sensors, `viam-server` can use the motion and navigation services to calculate how much power to send to each motor to get it to a point on the map.
```